tests/wpt/web-platform-tests/html/browsers/history/the-location-interface/location_origin.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  test(function () {
+    assert_equals(
+      location.origin,
+      location.protocol + '//' + location.host,
```

Comment:
hmm, host includes the port if the port is not default. So this passes the test because the test runs on port 8000, not a default port for http.

This is the same logic as https://github.com/servo/servo/blob/master/tests/wpt/web-platform-tests/websockets/opening-handshake/003.html

This does seem a little fragile but I assume we can't run a single test on port 80 so I don't know how we could test the default port scenario.

ideas?